src/components/billingRecord/IFXCalculateBillingMonth.vue

```diff
@@ -75,7 +75,7 @@ export default {
       return !this.usagesPreviouslyBilled() || this.recalculate
     },
     hasFinalBillingRecord(usage) {
-      return usage?.billingRecords.some((br) => br.currentState === 'FINAL')
+      return usage?.billingRecords?.some((br) => br.currentState === 'FINAL')
```

Comment:
Could `usage` also be `null`? If not, you don't need the first `?.` I doubt it will hurt anything, though.